app/src/main/java/com/example/nguyenxuansang98/studentmanagement/ui/MainContract.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.nguyenxuansang98.studentmanagement.ui
+
+import android.content.Context
+import com.example.nguyenxuansang98.studentmanagement.data.Student
+
+interface MainContract {
+
+    interface View{
+        fun addFalse(error:String)
+        fun addSuccess(success:String)
```

Comment:
View chỉ cần biết những hành vi: hiển thị, thông báo, cập nhật, ... 
Nó k cần biết logic ở đâu mà nó được xử lý ở dưới presenter 
```suggestion
        fun showMessage(msg: String)
```